.mergify.yml

```diff
@@ -0,0 +1,37 @@
+pull_request_rules:
+  - name: automatic deletion of pull-request branches after merge
+    conditions: []
+    actions:
+      # Takes no options, see <https://doc.mergify.io/examples.html#deleting-merged-branch>
+      delete_head_branch: {}
+
+  - name: automatic merge when CI passes and 2 reviews
+    conditions:
+      # Only pull-requests sent to the master branch
+      - base=master
+
+      # All Azure builds should be green:
+      - status-success=Uno.UI - CI
+
+      # CLA check must pass:
+      #- "status-success=license/cla"
+
+      # Do not automatically merge pull-requests that are labelled as do-not-merge
+      # see <https://doc.mergify.io/examples.html#id6>
+      - label!=do-not-merge/breaking-change
+      - label!=do-not-merge/work-in-progress
+
+      # Note that this only matches people with write / admin access to the repo,
+      # see <https://doc.mergify.io/conditions.html#attribute-list>
+      - "#approved-reviews-by>=2"
+
+      # Note: mergify cannot break branch protection rules
+    actions:
+      merge:
+        method: merge
+
+#  - name: automatic removal of possibly outdated reviews approvals or changes request when a pull request gets updated with new commits
```

Comment:
I think, for now, we should keep this option disabled as it provides the ability for folks to squash their commits after approvals have been done.